infrastructure/terraform/accounts/modules/account/aws/variables.tf

```
@@ -0,0 +1,10 @@
+variable "prefix" {}
+
+variable "principals" {
+  type = list
```

better:

```suggestion
  type = list(string)
```

This will give better error messages if you do something like `["${var.principals}"]`, which is a bad habit from Terraform 0.11. Plus stronger type hints are better!